Add relationships, filters and sparse-fieldsets pages (#9)
MewsSystems/gitbook-pos-api: essential-guide/authentication.md

# Authentication

> The Mews POS API uses bearer authentication, also known as token authentication. When you make an API call, you need to include your API key in the `Authorization` header in the following format:

Reviewer: ```suggestion
The __Mews POS API__ uses bearer authentication, also known as token authentication. When you make an API call, you need to include your API key in the `Authorization` header in the following format:
```

```
Bearer <your API key>
```

> The `<your api key>` is a placeholder for your key which should have `sk_test` prefix for the test environment and `sk_live` prefix for the production environment.

Reviewer: ```suggestion
`<your api key>` is a placeholder for your key which should have `sk_test` prefix for the test environment and `sk_live` prefix for the production environment.
```

> It is important to keep your API keys safe. Avoid sharing your secret API keys in publicly accessible locations like GitHub or client-side code. 

Reviewer: I suggest a breakout box here:
```suggestion
> **Important**: It is important to keep your API keys safe. Do not share your secret API keys in publicly accessible locations like GitHub or client-side code. 
```
